chain-signatures/node/src/protocol/mod.rs

```diff
@@ -381,3 +385,44 @@ fn node_version() -> i64 {
     };
     (rc_num + version.patch * 1000 + version.minor * 1000000 + version.major * 1000000000) as i64
 }
+
+
+fn update_system_metrics() {
+    let mut system = System::new_all();
+
+    // Refresh only the necessary components
+    system.refresh_all();
+
+    // Update CPU usage metric
+    let cpu_usage = system.global_cpu_usage() as i64;
+    crate::metrics::CPU_USAGE_PERCENTAGE
+        .with_label_values(&["global"])
```

Comment:
better also add account_id to the label values. You'd likely need to pass in the account_id in the update_system_metrics() to be able to access it. The account_id value is available here: https://github.com/near/mpc/blob/d8d8e386f412b7870454a293e5b101c7ab0668b7/chain-signatures/node/src/protocol/mod.rs#L204

Reply:
Should be fixed now I think